### PR TITLE
Adding gpx script

### PR DESCRIPTION
**Pull request by drillinP**
A Javascript script that allows you to add layers in gpx format
in Lizmap, and then edit them.

### CODE REVIEW

File: edit_gpx/GPX.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        strokeWidth: selectStrokeWidth
+    })
+    });
+    map = map;
+
+    vectors = new OpenLayers.Layer.Vector('test', {
+      styleMap: myStyles
+    });
+    map.addControl(new OpenLayers.Control.LayerSwitcher());
+    map.addControl(new OpenLayers.Control.MousePosition());
```
Comment (rldhont) [inline edit_gpx/GPX.js]:
```suggestion

```

File: edit_gpx/GPX.js [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        strokeColor: selectStrokeColor,
+        strokeWidth: selectStrokeWidth
+    })
+    });
+    map = map;
+
+    vectors = new OpenLayers.Layer.Vector('test', {
+      styleMap: myStyles
+    });
+    map.addControl(new OpenLayers.Control.LayerSwitcher());
```
Comment (rldhont) [inline edit_gpx/GPX.js]:
```suggestion

```

File: edit_gpx/GPX.js [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+    //var layer = getLayer(layers[0]);
+    //var features = layer.features();
+    if(gpxLayer === undefined){
+      alert('Veuillez importer un GPX en premier');
+    }else{
+      var features = gpxLayer.features;
+      for(var i in features){
+        var item = features[i];
+        if(item.geometry instanceof OpenLayers.Geometry.LineString){
+          console.log("ok line");
```
Comment (rldhont) [inline edit_gpx/GPX.js]: clean up

File: edit_gpx/GPX.js [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+
+      element.click();
+  }else{
+    alert("Export impossible Veuillez saisir un nom de fichier !")
+  }
+  });
+
+  $("#getgeom").click(function(){
+    //var layer = getLayer(layers[0]);
+    //var features = layer.features();
```
Comment (rldhont) [inline edit_gpx/GPX.js]: clean up

**Comment by josemvm** [comment]
hi @drillinP 

very nice!!!

is it possible to improve so that you can erase a vertex?

thanks

File: edit_gpx/GPX.js [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+    }),
+    "select": new OpenLayers.Style({
+        pointRadius: pointRadius,
+        fillColor: selectFillColor,
+        strokeColor: selectStrokeColor,
+        strokeWidth: selectStrokeWidth
+    })
+    });
+
+    vectors = new OpenLayers.Layer.Vector('default name', {
```
Comment (rldhont) [inline edit_gpx/GPX.js]: Can you named the OpenLayers vector layer as `gpx` ?
```suggestion
    vectors = new OpenLayers.Layer.Vector('gpx', {
```

**Comment by mdouchin** [comment]
Feature added to master (manually). I close this PR. Thanks all for your work, comments and reviews

**Pull request closed without merging**